examples/jenkins/application-template.json

```diff
@@ -314,7 +349,27 @@
     }
   ],
   "parameters": [
+     {
+      "name": "DEPLOYMENT_REQUEST_SIZE",
+      "description": "Memory rquested on startup from OpenShift Quota System",
+      "value": "64Mi"
+    },
     {
+      "name": "DEPLOYMENT_LIMIT_SIZE",
+      "description": "Maximum amount of memory allowed by OpenShift Quota System",
+      "value": "128Mi"
+    },
+     {
+      "name": "BUILD_REQUEST_SIZE",
+      "description": "Memory rquested on startup from OpenShift Quota System",
+      "value": "32Mi"
+    },
+    {
+      "name": "BUILD_LIMIT_SIZE",
```

Comment:
Don't add build limits or request unless we know that the image will not run without at least X.